### DISCUSSION TITLE
Feature Request: FidelityFX Super Resolution for viewport/rendering

### DISCUSSION THREAD

**Discussion by OneDayCreations** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#340] [Ideas]
FSR seems like an ideal way to help boost productivity being able to render at a lower resolution and upscale would save a lot of time during the production period when you are trying to just get a good idea of what you are working with. Even if it is just a pass that happens post render for compositing like denoising. Between this and using a lower level of raytracing in viewport I think you could really boost the amount of creative decisions you make in a short time =) 

If it can work in real-time for gaming with such big boosts to render speed there, surely it could have the same effect when rendering in production?

**Comment by bsavery** [comment]
Hi there.  So we actually do something currently with the "viewport upsampling" (look in the viewport sampling properties).  We use a slightly different technique than the FidelityFX but might enable that one in the future.

For final renders I don't think it makes sense though as there can be image differences.
